.env.example

```
@@ -1,24 +1,48 @@
+POSTGRES_USER=postgres
+DATABASE_URL="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:${POSTGRES_PORT}/${POSTGRES_DB}"
 
+# Redis
 REDIS_PASSWORD=example-redis-password
-REDIS_URL="redis://default:${REDIS_PASSWORD}@localhost:6379/0"
+REDIS_PORT=6379
+REDIS_URL="redis://default:${REDIS_PASSWORD}@localhost:${REDIS_PORT}/0"
+
+# Storage
+STORAGE_BACKEND=s3
+MINIO_API_PORT=9000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_API_PORT key should go before the STORAGE_BACKEND key

(UnorderedKey)